bin/ssl.sh

```diff
@@ -8,7 +8,7 @@ set -x  # be chatty and show the lines we're running
 # that are mysterious and not understood, so let
 # bail early if we detect that's the case
 ENGINE_OPENSSL=`openssl version | awk '{print $(1)}'`
-if [ "$ENGINE_OPENSSL" == "LibreSSL" ]
+if [ "$ENGINE_OPENSSL" = "LibreSSL" ]
```

Comment:
Noticed some shell errors on github actions when running on ubuntu. Saw that the `==` syntax isn't cross-shell compatible. Same with `-a` vs `-e`

Reply:
These changes fixed those errors

Reply:
good catch. bash... is not my favorite haha. *watches dev cred decrement by 5*

Reply:
Oh this was def a TIL for me